docker/centos7/Dockerfile

```diff
@@ -153,6 +159,8 @@ RUN source /opt/rh/devtoolset-8/enable && \
         -DCMAKE_BUILD_TYPE=Release \
         -G Ninja \
         -Wno-dev \
+        # Enable RTTI for autofdo
```

Comment:
remove the comment, or add it to the comment block at the beginning of the LLVM build/install